CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ option(WANT_IMAGE "Enable image load/save addon" on)
 if (NOT IPHONE)
     option(WANT_IMAGE_JPG "Enable JPEG support in image addon" on)
     option(WANT_IMAGE_PNG "Enable PNG support in image addon" on)
+    option(WANT_IMAGE_WEBP "Enable WebP support in image addon" on)
```

Comment:
Is libwebp not supported on iOS? Otherwise this should go outside of the iOS define.

Reply:
I don't really understand this construct to be honest and left it there as an uninformed guess. Why are JPG and PNG wrapped in "NOT IPHONE" conditional? I see that Allegro on iOS uses system provided facilities to load image files, but it does so on macOS, Android and Windows as well. What's so different about the iPhone?